### PR TITLE
Add "funding" key in study and add selection criteria' translations

### PR DESCRIPTION
**Pull request by Rima-B**

### CODE REVIEW

File: mica-core/src/main/resources/taxonomies/mica-study.yml [hunk 1]
```diff
@@ -1509,7 +1508,33 @@
...
       "en": "No Newborn have been recruited."
-      "fr": "Des nouveaux nés non pas été recrutés."
+      "fr": "Des nouveaux-nés non pas été recrutés."
+
+- "name": "populations-selectionCriteria-twins"
+  "title":
+    "en": "Twins"
+    "fr": "Jumeaux"
+  "description":
+    "en": "The recruited individuals are Newborn."
```
Comment (meek0) [inline mica-core/src/main/resources/taxonomies/mica-study.yml]: Twins instead of newbown

**Comment by meek0** [comment]
Website's change from type string to localizedString causes an error for old studies, new ones are okay.

File: mica-core/src/main/resources/config/study-form/schema.json [hunk 2]
```diff
@@ -2,8 +2,14 @@
   "type": "object",
   "properties": {
     "website": {
-      "title": "t(website)",
-      "type": "string"
+      "type": "object",
+      "format": "localizedString",
```
Comment (ymarcon) [inline mica-core/src/main/resources/config/study-form/schema.json]: Not sure it is a good idea to have a localized website in the generic.

File: mica-core/src/main/resources/config/study-form/definition.json [hunk 3]
```diff
@@ -189,7 +189,7 @@
           {
             "key": "access",
             "notitle": true,
```
Comment (ymarcon) [inline mica-core/src/main/resources/config/study-form/definition.json]: notitle = bad practice

File: mica-core/src/main/resources/config/study-form/definition.json [hunk 4]
```diff
@@ -189,7 +189,7 @@
...
-            "description": "t(study_taxonomy.vocabulary.access.description)",
+            "description": "t(study.access.for)",
```
Comment (ymarcon) [inline mica-core/src/main/resources/config/study-form/definition.json]: Not consistent with the term titles that follow

**Pull request merged**